test/test_manifest.json

```diff
@@ -389,6 +389,12 @@
        "rounds": 1,
        "type": "load"
     },
+    {  "id": "bug854315",
+       "file": "pdfs/bug854315.pdf",
+       "md5": "675754c07f71c068d2997905a5456f05",
+       "rounds": 1,
+       "type": "eq"
+    },
```

Comment:
This needs `"annotations": true` too I think since otherwise those are not rendered right now.

Reply:
I'm probably missing something trivial here, but shouldn't signatures render *automatically* on the canvas now?
Given this code: https://github.com/mozilla/pdf.js/blob/86374d4f75ef1ac599ee3a1431311d614075faed/src/core/document.js#L361-L407

---

Again I might be way off, but that `annotations: true` is intended for things normally rendered in the AnnotationLayer (in the viewer) which shouldn't apply here!?

Reply:
I've *just* tested locally, and a "normal" `eq`-test seems sufficient here!

Reply:
@Snuffleupagus I just tested locally too and same result.

Reply:
Whoops, that's indeed true. In that case it's indeed not necessary.